src/server/fetcher.ts

```diff
@@ -319,8 +320,7 @@ export class Fetcher<TOutput extends Response> {
         {
           ...this.config.httpOptions(),
           [customFetch]: (url: string, options: any) => {
-            const tmpRequest = new Request(url, options);
-            return this.config.fetch(tmpRequest);
+            return this.config.fetch(url, options);
```

Comment:
We no longer need to create a new Request object here because the fetch is compatible with native fetch.